packages/react-charting/src/components/CommonComponents/ErrorBoundary.base.tsx

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import * as React from 'react';
+import { classNamesFunction, getId } from '@fluentui/react/lib/Utilities';
+import { IErrorBoundaryProps, IErrorBoundaryStyles } from './ErrorBoundary.types';
+import { ErrorBadgeIcon, ErrorIcon, StatusErrorFullIcon } from '@fluentui/react-icons-mdl2';
+
+export enum ErrorCodes {
+  NoData = 'No data',
+  GeneralError = 'General error',
+}
+
+interface IErrorBoundaryState {
+  hasError: boolean;
+  errorMsg: string;
+}
+const getClassNames = classNamesFunction<IErrorBoundaryProps, IErrorBoundaryStyles>();
+
+export class ErrorBoundaryBase extends React.Component<IErrorBoundaryProps, IErrorBoundaryState> {
+  private _defaultErrorMsg = "Something went wrong and we couldn't get the page to display";
+  private _classNames;
+  public static getDerivedStateFromError(error: Error) {
+    return { hasError: true, errorMsg: error.message };
+  }
+
+  constructor(props: IErrorBoundaryProps) {
+    super(props);
+    this.state = {
+      hasError: false,
+      errorMsg: '',
+    };
+    this._classNames = getClassNames(this.props.styles!, {
+      hasEmptyState: this.props.hasEmptyState,
+      theme: this.props.theme!,
+      width: this.props.width,
```

Comment:
where are you using the width and height classes

Reply:
ErrorBoundary.styles.ts